PSKoans/Koans/Constructs and Patterns/AboutPSObjects.Koans.ps1

```diff
@@ -91,13 +101,14 @@ Describe 'PSObject' {
         Add-Member -InputObject $Object -MemberType NoteProperty -Name 'TestProperty' -Value __
 
         $Object.TestProperty | Should -Be 12
-        '__' | Should -Be $Object.PSObject.Properties['TestProperty'].MemberType
+        '____' | Should -Be $Object.PSObject.Properties['TestProperty'].MemberType
 
         $PropertyTypes = $Object.PSObject.Properties |
             Group-Object -Property MemberType |
-            Select-Object -ExpandProperty Name
+            Select-Object -ExpandProperty Name |
+            Sort-Object
 
-        @( '__', '__' ) | Should -Be $PropertyTypes
+        @( '____', '____', '____' ) | Should -Be $PropertyTypes
```

Comment:
The above code to get PropertyTypes only gets me two types of properties. Is it getting 3 for you?